ui/src/main.js

```diff
@@ -8,57 +9,66 @@ import routes from "./routes/routes";
 import en from "./translations/en.json";
 import stores from "./stores/store";
 import {setupTenantRouter} from "./composables/useTenant";
+import {apiUrlWithoutTenants} from "./override/utils/route";
 
 
 const app = createApp(App)
 
-initApp(app, routes, stores, en).then(({store, router, piniaStore}) => {
+const validateCredentials = async (credentials) => {
+    const [username, password] = atob(credentials).split(":")
+    
+    const response = await axios.post(`${apiUrlWithoutTenants()}/basicAuth`, {
```

Comment:
Does this mean you will validate credentials even if there is no request to the API?